website/content/en/docs/best-practices/pod-security-standards.md

```diff
@@ -393,7 +387,23 @@ var _ = Describe("my operator test", func() {
 })
 ```
 
- 
+### After following the recommendations to be restricted my workload is not running (CreateContainerConfigError). What should I do?
+
+If you are checking errors like `Error: container has runAsNonRoot and image has non-numeric user` 
+or `container has runAsNonRoot and image will run as root` that means that the image used has not an user defined, i.e.:
+
+```shell
+USER 65532:65532 
+OR
+USER 1001
+```
+
+Due to RunAsNonRoot field being set to true and we need to force the user in the
+container to a non-zero numeric user in order. However, if you are looking to provide solution 
+for K8s vendors like OpenShift be aware that your workload can not qualify for restricted-v2 SCC 
+if you set this value. In this way, if you are looking for distribute your Operator not only
+on vanilla Kubernetes would be recommended you change the image by setting a user. Otherwise,
+you can define the user via the security context configuration. (i.e. `RunAsUser: &[]int64{1000}[0],`).
```

Comment:
Some rewording:
```suggestion
Due to the `RunAsNonRoot` field being set to `true`, we need to force the user in the
container to a non-zero numeric user.
It is recommended that the images used by your operator have a non-zero numeric user set in the image itself (similar to the example above).
In vanilla Kubernetes you can also define the user via the security context configuration. (i.e. `RunAsUser: &[]int64{1000}[0],`). 
It is important to note that defining the `RunAsUser` field will not work on K8s vendors like OpenShift, as the `RunAsUser` field violates the restricted-v2 SCC used by OpenShift. In this case the user will need to be defined in the image. 
```

Reply:
I think we can inded make it better see: https://github.com/docker-library/memcached/pull/79#issuecomment-1181044418 it seems a good practice for docker as well